.github/workflows/unit-tests.yaml

```diff
@@ -22,7 +22,11 @@ jobs:
           fetch-depth: 0
           set-safe-directory: ${{ env.GITHUB_WORKSPACE }}
       - name: Install packages
-        run: dnf install -y gcc krb5-devel make glibc
+        run: dnf install -y gcc krb5-devel make glibc python3-rpm
```

Comment:
there used to be an error with `Failed to import system RPM module. Make sure RPM Python bindings are installed on your system.` on https://github.com/openshift-eng/art-tools/actions/runs/11624887515/job/32374119155 so import rpm in python rely on `python3-rpm` rpm instead of rpm lib in requirements.txt